content/articles/integrating-chart.js-in-django/index.md

```diff
@@ -0,0 +1,305 @@
+Chart.js is an open-source HTML5 based javascript chart and renders in HTML5 canvas. It basically helps in visualizing data using javascript. 
+
+In this tutorial, we'll be looking at how to integrate Chart.js in Django by creating a simple project.
+
+### Prerequisites
+To follow along with this tutorial, you'll need [Python3](https://www.python.org/downloads/) installed on your machine.
+
+A little understanding of Django would help the reader to follow along better.
+
+### Table of contents
+1. [Creating and setting up a new Django project](#creating-and-setting-up-a-new-django-project)
+2. [Creating a model](#creating-a-model)
+3. [Creating a view](#creating-a-view)
+4. [Creating templates](#creating-templates)
+5. [Conclusion](#conclusion)
+
+### Creating and setting up a new Django project
+Let's start by creating a new virtual environment.
+
+A virtual environment allows you to create different spaces on your computer, with a different set of libraries and versions.
+
+By creating a virtual environment, you'll be able to separate the necessary library installation for a project, without having to install them globally.
+
+Now, create a virtual environment `env` by running the command below:
+
+```bash
+$ python -m venv env
```

Comment:
You could remove `$` from these code snippets.

This would allow the readers to copy-paste the codes easily

Reply:
Do this for all the below code snippets